frontend/views/components/SliderContinuous.vue

```diff
@@ -0,0 +1,150 @@
+<template lang='pug'>
+.wrapper
+  label.label(:for='`range${uid}`') {{ label }}
+
+  span.marks
+    span.edge(aria-hidden='true') {{ min }}
+
+    span.slider(:style='ephemeral.styleVars')
+      input.sInput(
+        type='range'
+        :id='`range${uid}`'
+        :min='min'
+        :max='max'
+        :value='value'
+        @input='handleChange'
+      )
+      output.sOutput(
+        :for='`range${uid}`'
+        :style='ephemeral.outputStyle'
+      ) {{ value + unit }}
+
+    span.edge(aria-hidden='true') {{ max }}
+</template>
+
+<script>
+export default {
+  name: 'SliderContinuous',
+  props: {
+    /** Unique id to connect form and label */
+    uid: {
+      type: String,
+      required: true
+    },
+    label: String,
+    min: [String, Number],
+    max: [String, Number],
+    unit: String,
+    value: [String, Number]
```

Comment:
Just curious, does it make sense for min/max/value to be either a String or a Numbers?

Math is performed on them directly below, so I think they should be numbers?

Reply:
Technically for HTML, every attribute is a String, not a Number.

We could pass as a number, but HTML would interpret it and return as a String. So this go-and-back will return a prop type error eventually. So we need to accept both types.

Reply:
@sandrina-p for Vue.js to pass numbers you do `:value='3'` (note the `:`). So can you change it to do it that way?

Reply:
Maybe I didn't explain myself well. You can pass as a Number, but HTML will retrieve a String. `@input(e => typeof e.target.value) // String`

So, that's why we need both. So that when the prop goes up to the parent and is updated, it can be a String or Number.

Reply:
*sigh*... ok, sorry, didn't understand. OK thanks!